layer/clients/label.py

```diff
@@ -65,3 +81,13 @@ def get_labels_attached_to(
             self._service.GetLabelsAttachedToEntity(request=request)
         )
         return set(map(lambda l: l.name, resp.label))
+
+    def get_labels_attached_to_run(
+        self,
+        run_id: UUID,
+    ) -> Set[str]:
+        request = GetLabelsAttachedToRunRequest(run_id=RunId(value=str(run_id)))
```

Comment:
Same as above you can use the proto mapper: https://github.com/layerai/sdk/blob/main/layer/clients/protomappers/runs.py#L11